Client/Frontend/Strings.swift

```diff
@@ -797,3 +797,15 @@ extension Strings {
 extension String {
     public static let FxAQRCode_Instructions = NSLocalizedString("fxa.qr-scanning-view.instructions", value: "Scan the QR code shown at firefox.com/pair", comment: "Instructions shown on qr code scanning view")
 }
+
+//Today Widget Strings - [New Search - Private Search]
+extension String {
```

Comment:
I thought we added these strings? Didn't we?

Reply:
Amy made changes to the normal search button text, I'll make it in a separate PR